.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
     -   id: cfn-python-lint
         files: templates/.*\.(json|yml|yaml)$
 -   repo: https://github.com/Lucas-C/pre-commit-hooks
-    rev: v1.1.9
+    rev: v1.1.6
```

Comment:
is there a reason why we are going backwards?

Reply:
There was an issue that was specific to my environment not letting me use v1.1.9.

The repo owner very helpfully [assisted me](https://github.com/Lucas-C/pre-commit-hooks/issues/17) in fixing the issue.